sdk/src/main/java/com/mapbox/maps/MapView.kt

```diff
@@ -261,10 +264,11 @@ open class MapView : FrameLayout, MapPluginProviderDelegate, MapControllable {
   /**
    * Set new maximum FPS for map rendering.
    *
-   * @param maximumFps new maximum FPS value that must be > 0.
+   * @param fps new maximum FPS value that must be > 0 and less than 1000.
+   * When setting this value higher than screen physically supports - max possible screen FPS rate will be used.
    */
-  override fun setMaximumFps(@IntRange(from = 1L, to = Int.MAX_VALUE.toLong()) maximumFps: Int) {
-    mapController.setMaximumFps(maximumFps)
+  override fun setMaximumFps(@IntRange(from = 1L, to = MAX_POSSIBLE_FPS) fps: Int) {
```

Comment:
can we keep remove the `MAX_POSSIBLE_FPS` and use `Int.MAX_VALUE` instead? it feels pretty weird to have 1000 here, actually tested with `Int.MAX_VALUE` and the map renders pretty ok on my device.